R/execution.r

```diff
@@ -133,6 +134,14 @@ execute = function(request) {
     assign('quit', quit, envir = .GlobalEnv)
     assign('q',    quit, envir = .GlobalEnv)
     
+    send_error_msg <- function(msg){
```

Comment:
It’s a send_response call with most fields filled in. I always create small helpers like that to reduce complexity